github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -375,9 +372,6 @@ presubmits:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
-      - name: unprivileged-token
-        secret:
-          secretName: unprivileged-oauth-token
```

Comment:
I think we should not touch the job config for this job, it actually uses the unprivileged token to not cause any interference when testing the prow instance within the kind cluster IIRC.
Especially not fetching GitHub events for the real prow instance (which is what happened when we first tried this accidentally with the same token as the production prow uses) and also not messing with PR status then.